metrics/errors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'enables JavaScript Sentry error metrics',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uuid', 'sampleRate'],
+  independentVariables: ['sentryEnabled'],
+  eligibilityFunction: function (subject) {
+    return !! subject.sampleRate && subject.uuid && this.bernoulliTrial(subject.sampleRate, subject.uuid)
```

Comment:
> Would it make sense to skip /config and just define the sample rate here and update as needed?

Sample rate depends on the environment, Development should be 100%, Prod will have a different sample rate (I think we are going for 30% for starters).